.travis/linux_apisix_current_luarocks_runner.sh

```diff
@@ -40,19 +40,20 @@ do_install() {
     cd ..
     rm -rf luarocks-2.4.4
 
-    ./utils/install-etcd.sh
+    # ./utils/install-etcd.sh
+    ./utils/install-etcd-client.sh
```

Comment:
I think we don't need etcd client.

Reply:
https://github.com/apache/apisix/blob/master/t/core/etcd-auth.t#L29

in test case, need it.

Reply:
OK